README.md

```diff
@@ -1,25 +1,25 @@
 # placement-exams
 
 Designed to support University of Michigan placement exams administered through the Canvas LMS,
-the in-development placement-exams application will collect the students' scores from multiple exams
-and send them to M-Pathways so the registrar can grant course enrollment privileges. The UM API Directory
-will be used both to access the Canvas API and send scores to M-Pathways.
+the placement-exams application collects students' scores for multiple exams
+and sends them to M-Pathways so the registrar can grant course enrollment privileges. The UM API Directory
+is used both to access the Canvas API and send scores to M-Pathways.
 
 ## Development
 
 ### Pre-requisities
 
-The sections below provide instructions for configuring, installing, using, and changing 
-the application. Depending on the environment you plan to run the application in, you may 
+The sections below provide instructions for configuring, installing, and using the application.
+Depending on the environment you plan to run the application in, you may
 also need to install some or all of the following:
 
-* [Python 3.8](https://docs.python.org/3/)
-* [MySQL](https://dev.mysql.com/doc/)
-* [Docker Desktop](https://www.docker.com/products/docker-desktop)
-* [OpenShift CLI](https://docs.openshift.com/enterprise/3.1/cli_reference/get_started_cli.html)
+*   [Python 3.8](https://docs.python.org/3/)
```

Comment:
the link points to current Python 3.9 version. Maybe use https://docs.python.org/3.8/ instead?

Reply:
Good idea, thanks for the catch.